docs/intro/tutorial.rst

```diff
@@ -819,6 +819,60 @@ will only visit URLs from the ``humor`` tag, such as
 
 You can :ref:`learn more about handling spider arguments here <spiderargs>`.
 
+
+Common Errors and Troubleshooting Tips
+======================================
+
+**Creating Your First Scrapy Project**
+
+Error: ``Command 'scrapy startproject' not found``
+
+* Cause: Scrapy might not be installed or properly configured in your environment.
+* Solution: Ensure that Scrapy is installed. You can install it via pip by running 
+  ``pip install scrapy``. If you're working in a virtual environment, make sure the 
+  environment is activated before running the command.
```

Comment:
Maybe this is better covered as a ``.. tip::`` section right after the first execution of the command in the tutorial.

Also, I would omit `startproject` when describing the issue, as usually the error is specifically about the `scrapy` part. If `startproject` is mentioned in the error, `scrapy` is most likely installed.